jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecr/actions/PushTagToRepositoryAction.kt

```diff
@@ -150,20 +209,155 @@ internal class PushToEcrDialog(
                     text = value.tag ?: value.imageId.take(15)
                 }
             )
-                .sizeGroup(selectorGroup)
+                .constraints(grow)
                 .withErrorOnApplyIf(message("ecr.image.not_selected")) { it.selected() == null }
         }
+    }
 
-        row(message("ecr.push.remoteTag")) {
-            textField(::remoteTag)
-                .sizeGroup(selectorGroup)
-                .withErrorOnApplyIf(message("ecr.tag.not_provided")) { it.blankAsNull() == null }
+    private fun dockerfileConfigurationSelectorPanel() = panel {
```

Comment:
curious why this is a function if we call it in init anyway?

Reply:
no particular reason, it was just getting too hard to follow as a single block